sdk-autogen/pom.xml

```diff
@@ -12,7 +12,6 @@
 
   <artifactId>dapr-sdk-autogen</artifactId>
   <packaging>jar</packaging>
-  <version>1.1.0-SNAPSHOT</version>
```

Comment:
What happens when you run this script? `https://github.com/dapr/java-sdk/blob/master/scripts/update_sdk_version.sh`

We use that for our release process to automatically update the version.

Reply:
> What happens when you run this script? `https://github.com/dapr/java-sdk/blob/master/scripts/update_sdk_version.sh`
> 
> We use that for our release process to automatically update the version.

It says `update_sdk_version.sh: line 5: $1: unbound variable`.

The environment variable is missing here `DAPR_JAVA_SDK_VERSION=$1`.

Reply:
Pass a value, like "1.2.0"

Reply:
> Pass a value, like "1.2.0"

```
Limbo@DESKTOP-GLSB8CF MINGW64 /d/gitProject/java-sdk/scripts (master)
$ ./update_sdk_version.sh
+ DAPR_JAVA_SDK_VERSION=1.2.0
+ mvn versions:set -DnewVersion=1.2.0
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.291 s
[INFO] Finished at: 2021-04-20T07:18:30+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:versions-maven-plugin:2.8.1:set (default-cli): Goal requires a project to execute but there is no POM in this directory (D:\gitProject\java-sdk\scripts). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException
```

Works after I add ```cd ..``` before ```mvn versions```